### DISCUSSION TITLE
Bug: Everything is deleted when none is selected in Properties [3.00]

### DISCUSSION THREAD

**Discussion by malvarco** [transmission/transmission#4373] [Q&A] [answered]
### What is the issue?

This is a very annoying bug.
**Steps to Reproduce the Bug;**
1) Add .torrent or magnet link. Let's try this one as an example:
https://archive.org/download/Batman201-250/Batman201-250_archive.torrent

2) Go to torrent properties and, since I don't want all files, I want to uncheck all files, so then I could check only the files I want:
![1b](https://user-images.githubusercontent.com/4598007/207924216-9e984f07-d68c-42da-8ccf-c9d93a5d5e8d.jpg)

**Expected Result**
Show all files unchecked, so I can check only the files I want.

**Actual Result**
All files disappear, I can't check anything.
But what's even more annoying is that I can't try to download the files again, because I get a message error saying the file is duplicated:
![1a](https://user-images.githubusercontent.com/4598007/207924213-5acbdda4-7d9b-4490-ada9-98249015e86b.jpg)

### Which application of Transmission?

Qt app on Windows

### Which version of Transmission?

3.00 stable

**Comment by ckerr** [comment, marked as answer]
The issue is that the main window was filtered by "Downloading" when you clicked to not download any files in that torrent. So Transmission figured "hm, not downloading anything here" which removed it from the main window's list due to that filter, and _that_ in turn caused the properties dialog to be cleared. :dizzy: 

If you chane the "Show:" filter to also show the torrents that aren't being downloaded, you should be able to find your torrent and re-select the files you wanted.
